k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/sample_traffic/nginx-traffic/nginx-traffic-sample.yaml

```diff
@@ -77,24 +77,30 @@ spec:
 
 ---
 
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Can you drop a reference for where this apiVersion comes from? Just so we can keep that in the PR for review

Reply:
done